Coding Tracker project

LucianoNicolasArrieta opened the pull request.

TheCSharpAcademy reviewed (changes requested):
Hey there! The project is looking good but I'm getting this error: 

![image](https://user-images.githubusercontent.com/108711476/233885726-dc8cd21e-d3b5-4b08-a021-31ae4d1c69eb.png)

LucianoNicolasArrieta commented:
Hi! fixed. I don't really know what happened because it works with some dates but with others not, i just split the method that validates the end date into two steps and it seems now it's working.

TheCSharpAcademy commented:
@LucianoNicolasArrieta hey there! I'm approving since all requirements are met but I'm not being able to see the reports: 

![image](https://user-images.githubusercontent.com/108711476/234427779-a40f1041-966e-46d5-9c66-883e1f085a69.png)

Also the app breaks if I don't have any records: 

![image](https://user-images.githubusercontent.com/108711476/234427946-ebb35338-d899-4d50-ad0a-19aec6d6cec3.png)
